src/store/wallet/interface.ts

```diff
@@ -4,7 +4,7 @@ import type { AnyJson } from "@polkadot/types/types/codec";
 import { ActionType } from "../../app/types/enum";
 
 interface TokenData {
-  balance: AnyJson;
+  balance: number;
```

Comment:
Are you sure that is this type correct? I remember that @ra79sta said something about balance type, but I'm not sure.

Reply:
this is okk, balance here will always be a number, i check with Kaliman